www/js/app.js

```diff
@@ -416,6 +419,16 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         // Do the necessary to enable or disable the Service Worker
         setContentInjectionMode(this.value);
     });
+    document.getElementById('useCanvasElementsCheck').addEventListener('change', function () {
+        if (this.checked) {
+            settingsStore.removeItem('useCanvasElements');
+            params.useCanvasElements = uiUtil.determineCanvasElementsWorkaround();
+            this.checked = params.useCanvasElements;
```

Comment:
So it means that there are cases where we would refuse that the user checks this? If so, we should tell the user why

Reply:
Yes, user can only turn this off if the app turned it on. Not vice versa.